torch/_torch_docs.py

```diff
@@ -2971,6 +2971,72 @@ def parse_kwargs(desc):
             [ 2.]])
 """)
 
+add_docstr(torch.frobenius_norm,
+           r"""
+.. function:: frobenius_norm(input) -> Tensor
+
+Returns the frobenius norm of the :attr:`input` tensor.
+
+.. math::
+    ||x||_{F} = \sqrt{tr(xx^{H})}
```

Comment:
Could you please change `H` to `T`. We don't have support for complex tensors yet.